src/styles/blocks/about-us.scss

```diff
@@ -62,7 +62,6 @@
       width: 100%;
 
       @include on-tablet {
-        padding-inline: 34px 0;
```

Comment:
padding is required there and the horizontal scroll problem still exists. It's not solve problem (and adds one more) - I'd revert or at least change it

Reply:
Coming up with changes. Thanks a lot. I just fixed it.